frontend/src/helpers/velocityAnimate.js

```diff
@@ -0,0 +1,74 @@
+import Velocity from 'velocity-animate'
+
+export const lineBeforeEnter = (el, x1, y1, x2, y2) => {
```

Comment:
Seemingly duplicate code in this file, could be refactored?

Reply:
We considered this but decided that it would reduce readability of code